tic-tac-toe/src/scripts/calculate-winner.js

```diff
@@ -0,0 +1,23 @@
+function calculateWinner(squares) {
+  const winCombinations = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6],
+  ];
+  for (let i = 0; i < winCombinations.length; i++) {
+    const [a, b, c] = winCombinations[i];
+    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+      
```

Comment:
remove these weird spaces in if-s

Reply:
I fixed this here and in other necessary places.
```suggestion
    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
      return squares[a];
    }
```